frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -1103,6 +1155,22 @@ int api2wire_weekdays(Weekdays raw) {
 
 // Section: wire2api
 
+Duration _wire2api_Chrono_Duration(dynamic raw) {
+  return Duration(milliseconds: _wire2api_i64(raw));
+}
+
+DateTime _wire2api_Chrono_Local(dynamic raw) {
+  return DateTime.fromMillisecondsSinceEpoch(_wire2api_i64(raw), isUtc: false);
+}
+
+DateTime _wire2api_Chrono_Naive(dynamic raw) {
+  return DateTime.fromMillisecondsSinceEpoch(_wire2api_i64(raw), isUtc: true);
+}
+
+DateTime _wire2api_Chrono_Utc(dynamic raw) {
+  return DateTime.fromMillisecondsSinceEpoch(_wire2api_i64(raw), isUtc: true);
+}
```

Comment:
here there's no `bridge_generated.io.dart` / `bridge_generated.web.dart` alternate representation and the generated code is bound on `IrTypeDelegate::Time(ir) if self.context.config.wasm_enabled => match ir { ... }` found at `frb_codegen/src/generator/dart/ty_delegate.rs`. So running frb codegen with `--wasm` or without `--wasm` will lead to 2 slightly different `frb_example/pure_dart/dart/lib/bridge_generated.dart` file.

Reply:
this is also probably the error shown in valgrind if it runs on the generated files from the commit.
<img width="772" alt="Screen Shot 2565-09-17 at 14 08 29" src="https://user-images.githubusercontent.com/21016014/190848862-befeab22-da25-4158-8a9a-e930d1e7debf.png">
This conversion error makes me think it uses `milliseconds` which the last code generation I ran and committed.

Reply:
>  but I need it to output chrono types in milliseconds or microseconds based on web or vm

For api_generates.rs: #[cfg(...)]

For generated .dart: if(kIsWeb)

Reply:
Indeed I tried already but `kIsWeb` is only available for flutter projects, not for dart.
I'll keep looking tomorrow.

Reply:
We can in fact stub `kIsWeb`: you can check its original definition which is a very fun read, or use `bool.fromEnvironment('dart.library.html')`.

Reply:
Oh yeah right @Desdaemon :smile: